project-euler/String-Calculator/string_calculator.rb

```diff
@@ -0,0 +1,8 @@
+class StringCalculator
+  def add(my_string)
+    return 0 if my_string.empty?
+    return my_string.to_i unless my_string.include? ','
```

Comment:
Essas linhas são necessárias? O que acontece se der um `''.split(',')`?

Reply:
Não são necessárias pois quando a string for vazia('') após utilizar o `.map(&:to_i).sum` o código ja atenderá todos os testes.